pixels-core/src/main/java/io/pixelsdb/pixels/core/writer/IntegerColumnWriter.java

```diff
@@ -55,8 +56,30 @@ public IntegerColumnWriter(TypeDescription type,  PixelsWriterOption writerOptio
     @Override
     public int write(ColumnVector vector, int size) throws IOException
     {
-        LongColumnVector columnVector = (LongColumnVector) vector;
-        long[] values = columnVector.vector;
+        if(vector instanceof LongColumnVector)
+        {
```

Comment:
Integer column writer should not use a LongColumnVector.

Reply:
We should split the integer column writer into int column writer and long column writer.